src/containers/Player.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
Write this as a dumb component first.

In general, JSX in `components/` and just JS in `containers/`. You're going to need to able to test the player's behavior via props, and you can't do that if it isn't in a separate component from the redux-connected container.